T/TempestRemap/build_tarballs.jl

```diff
@@ -0,0 +1,76 @@
+
+name = "TempestRemap"
+version = v"2.1.1"
+sources = [
+    ArchiveSource("https://github.com/ClimateGlobalChange/tempestremap/archive/refs/tags/v$(version).tar.gz",
+                  "f5ea21f82b358ba127550fc1f49e701cba0379d22ce030c274135109f678b980"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/tempestremap*
+
+autoreconf -fi
+mkdir -p build && cd build
+
+../configure \
+  --prefix=${prefix} \
+  --build=${MACHTYPE} \
+  --host=${target} \
+  --with-blas=blastrampoline \
+  --with-lapack=blastrampoline
+
+make -j${nproc} all
+make install
+
+install_license ../LICENSE
+"""
+
+# Note: We are restricted to the platforms that NetCDF supports, the library is Unix only
+platforms = [
+    Platform("x86_64", "linux"),
+    Platform("aarch64", "linux"; libc="glibc"),
+    Platform("x86_64", "macos"),
+] 
+platforms = expand_cxxstring_abis(platforms)
+
+products = Product[
+    LibraryProduct("libTempestRemap", :libTempestRemap),
+    ExecutableProduct("AnalyzeMap", :AnalyzeMap_exe),
+    ExecutableProduct("ApplyOfflineMap", :ApplyOfflineMap_exe),
+    ExecutableProduct("CalculateDiffNorms",  :CalculateDiffNorms_exe),
+    ExecutableProduct("CoarsenRectilinearData", :ApplyOfflineMap_exe),
+    ExecutableProduct("ConvertExodusToSCRIP", :ConvertExodusToSCRIP_exe),
+    ExecutableProduct("ConvertSCRIPToExodus", :ConvertSCRIPToExodus_exe),
+    ExecutableProduct("GenerateCSMesh", :GenerateCSMesh_exe),
+    ExecutableProduct("GenerateGLLMetaData", :GenerateGLLMetaData_exe),
+    ExecutableProduct("GenerateICOMesh", :GenerateICOMesh_exe),
+    ExecutableProduct("GenerateLambertConfConicMesh", :GenerateLambertConfConicMesh_exe),
+    ExecutableProduct("GenerateOfflineMap", :GenerateOfflineMap_exe),
+    ExecutableProduct("GenerateOverlapMesh", :GenerateOverlapMesh_exe),
+    ExecutableProduct("GenerateOverlapMesh_v1", :GenerateOverlapMesh_v1_exe),
+    ExecutableProduct("GenerateRLLMesh", :GenerateRLLMesh_exe),
+    ExecutableProduct("GenerateRectilinearMeshFromFile", :GenerateRectilinearMeshFromFile_exe),
+    ExecutableProduct("GenerateStereographicMesh", :GenerateStereographicMesh_exe),
+    ExecutableProduct("GenerateTestData",  :GenerateTestData_exe),
+    ExecutableProduct("GenerateTransectMesh",  :GenerateTransectMesh_exe),
+    ExecutableProduct("GenerateTransposeMap", :GenerateTransposeMap_exe),
+    ExecutableProduct("GenerateUTMMesh", :GenerateUTMMesh_exe),
+    ExecutableProduct("GenerateVolumetricMesh", :GenerateVolumetricMesh_exe),
+    ExecutableProduct("MeshToTxt", :MeshToTxt_exe),
+    ExecutableProduct("RestructureData", :RestructureData_exe),
+    ExecutableProduct("ShpToMesh", :ShpToMesh_exe),
+    ExecutableProduct("VerticalInterpolate", :VerticalInterpolate_exe),
+]
+
+dependencies = Dependency[
+    Dependency("libblastrampoline_jll"),
+    Dependency("NetCDF_jll"),
+    Dependency("HDF5_jll"),
+]
+
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+    julia_compat="1.6",
```

Comment:
Have you tried this package in Julia v1.6?  I presume this package uses LP64 (as most packages do by default)?  If so, chances are this will either error or segfault at runtime

Reply:
I thought this was a compatible 1.6 feature but as you said it is 1.7